learn/get-started/write-a-restful-api-with-ballerina.md

```diff
@@ -321,6 +240,89 @@ public type InvalidIsoCodeError record {|
 In this code:
 - As in the previous example, this resource also includes its own return types. However, the basic principle behind them is as the previous example. 
 
+## The complete code
+
+The below is the complete code of the service implementation.
+
+```ballerina
+import ballerina/http;
+
+service /covid/status on new http:Listener(9000) {
+
+    resource function get countries() returns CovidEntry[] {
+        return covidTable.toArray();
+    }
+
+    resource function post countries(@http:Payload CovidEntry[] covidEntries)
+                                    returns CreatedCovidEntries|ConflictingIsoCodesError {
+
+        string[] conflictingISOs = from CovidEntry covidEntry in covidEntries
+            where covidTable.hasKey(covidEntry.iso_code)
+            select covidEntry.iso_code;
+
+        if conflictingISOs.length() > 0 {
+            return <ConflictingIsoCodesError>{
```

Comment:
Unless you are using these explicit casts to make return types more clear to readers, they are no longer needed by the compiler
```suggestion
            return {
```

Reply:
In that case we can remove the cast.